docs/installation/ca/Installing_CA_with_ECC.md

```diff
@@ -27,74 +27,22 @@ Supported ECC key algorithms:
 CA Subsystem Installation
```

Comment:
Under the Overview above this section, the link to setting up DS doesn't seem to have a clear indication on where to find instruction for setting up DS running with ECC server cert and algorithms.  It might help if such info is more readily available to fit the subject of this doc.

Reply:
Actually in this test (and in all other CI tests so far) we're always using non-SSL connection to DS to keep things focused on PKI instead of DS, so the ECC is only used for PKI certs. I'm not familiar yet with ECC setup for DS, so maybe we can add it as a future CI enhancement?

Reply:
Ah, okay, I thought this was to be tested with ssl connection. Sure we could add that later.